### PR TITLE
Building A Faster & Accurate Search Engine with Transformers🤗 & Haystack

### PR DESCRIPTION
**Pull request by Shivanandroy**

### CODE REVIEW

File: users/shivanandroy/Building-a-faster-accurate-search-engine-with-transformers/ARTICLE.md [hunk 1]
```diff
@@ -0,0 +1,388 @@
+This article will let you build a faster and accurate Search Engine using `Transformers🤗`
```
Comment (abhishekkrthakur) [inline users/shivanandroy/Building-a-faster-accurate-search-engine-with-transformers/ARTICLE.md:1]: Can you add the title here?

File: users/shivanandroy/Building-a-faster-accurate-search-engine-with-transformers/ARTICLE.md [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+
+
+![Search Engine](https://shivanandroy.com/posts/dl/images/covid19.jpg)
+
+
+<br>
```
Comment (abhishekkrthakur) [inline users/shivanandroy/Building-a-faster-accurate-search-engine-with-transformers/ARTICLE.md:7]: remove html

**Review by abhishekkrthakur — CHANGES_REQUESTED** [review]
Added some small comments. HTML is not allowed currently

**Comment by abhishekkrthakur** [comment]
Please use our renderer to preview your article: https://datamuni.com/renderer

**Comment by Shivanandroy** [comment]
Fixed as suggested.

File: users/shivanandroy/Building-a-faster-accurate-search-engine-with-transformers/ARTICLE.md (hunk 1 above)
Comment (Shivanandroy) [reply users/shivanandroy/Building-a-faster-accurate-search-engine-with-transformers/ARTICLE.md:4]: Added title

**Pull request merged**